### PR TITLE
Feature/aag 2200 login functionality

### PR DESCRIPTION
**Pull request by gnhn**
- Update login functionality
- Modernise the app

### CODE REVIEW

File: app/src/main/java/aademo/superawesome/tv/awesomeadsdemo/activities/login/LoginActivity.kt [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_login)
+
+        val button = findViewById<ProgressButton>(R.id.loginButton)
+        button.setText("Login")
+        button.isEnabled = true
+        button.setOnClickListener {
+            val username = findViewById<EditText>(R.id.usernameEditText).text.toString()
```
Comment (thomas-james-1986) [inline app/src/main/java/aademo/superawesome/tv/awesomeadsdemo/activities/login/LoginActivity.kt:25]: We could upgrade to using view binding here? It's not essential though

Comment (gnhn) [reply app/src/main/java/aademo/superawesome/tv/awesomeadsdemo/activities/login/LoginActivity.kt:25]: Yes, it was my initial choice. But then when I do that all of the activities and fragments needed to be updated for the viewBinding at the same time. So, I wanted to keep the PR brief and don't know if it worths the efforts

Comment (thomas-james-1986) [reply app/src/main/java/aademo/superawesome/tv/awesomeadsdemo/activities/login/LoginActivity.kt:25]: Yes fair enough, I suppose this is more about rehabilitating the app as opposed to refactoring

**Review by thomas-james-1986 — APPROVED** [review]
Looks good!

**Pull request merged**